### ISSUE TITLE
[BUG] Connection refused by server

### ISSUE THREAD

**Issue by servebeer** [linuxserver/docker-netbootxyz#65]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Currently, I f I pxeboot a system on the same subnet as NetBootXYZ, I see this:
![Screenshot 2024-04-19 at 14 34 22](https://github.com/linuxserver/docker-netbootxyz/assets/8514341/94ee47a9-2737-44a5-9135-5f850e3275e9)

This used to work, but I haven't tried in awhile. It appears that the DHCP settings are correct. I then installed a TFTP client on a different system on the same subnet and tried to connect the IP address of netbootxyz at port 69, and I get this error: 

Connection attempt failed with "ECONNREFUSED - Connection refused by server".
Error: Could not connect to server

Is there a setting in netbootxyz that specifically allows certain IP addresses to connect?

### Expected Behavior

I should be able to pxe boot a system, on the same subnet as the netbootxyz docker image, and get to the NetBootXYZ install menu.

### Steps To Reproduce

See above...

### Environment

```markdown
Unraid 6.12.10
```

### CPU architecture

x86-64

### Docker creation

```bash
Used the Unraid App Store image
```

### Container logs

```bash
[migrations] started
[migrations] no migrations found
usermod: no changes
───────────────────────────────────────

      ██╗     ███████╗██╗ ██████╗
      ██║     ██╔════╝██║██╔═══██╗
      ██║     ███████╗██║██║   ██║
      ██║     ╚════██║██║██║   ██║
      ███████╗███████║██║╚██████╔╝
      ╚══════╝╚══════╝╚═╝ ╚═════╝

   Brought to you by linuxserver.io
───────────────────────────────────────

To support the app dev(s) visit:
netboot.xyz: https://opencollective.com/netbootxyz/donate

To support LSIO projects visit:
https://www.linuxserver.io/donate/

───────────────────────────────────────
GID/UID
───────────────────────────────────────

User UID:    99
User GID:    100
───────────────────────────────────────

[custom-init] No custom files found, skipping...
Connection to localhost (127.0.0.1)Connection to localhost 69 port [udp/tftp] succeeded!
 (127.0.0.1) 80 port [tcp/http] succeeded!
Invalid port "undefined" in environment variable WEB_APP_PORT. Using default port 3000 instead.
listening on *:3000
V_wpsFsAtsmDCHyKAAAB connected time=1713373909590
Connection to localhost (127.0.0.1) 3000 port [tcp/*] succeeded!
[ls.io-init] done.
c0bhux3j18EtFN_PAAAD connected time=1713443571015
isCCfCgWkT_EGKuzAAAF connected time=1713443573910
oBu4i5zClIi1wOwRAAAH connected time=1713443576362
7ElmW1UF1aRbrvUiAAAJ connected time=1713443578196
dgtchimkumm4YirzAAAL connected time=1713451259265
kh7nUCNe-7qsaiQyAAAN connected time=1713451263397
qSIaeZtFkRiLGuWtAAAP connected time=1713452754069
MquVX639AeRXDLJVAAAR connected time=1713453307142
BeNeiIjK_poIfUKHAAAT connected time=1713454708057
uLj20gk37czlYQdGAAAV connected time=1713455427333
7LKlxZO45Sf9v1iFAAAX connected time=1713529964310
YwZEw8Ko8yK_VxbZAAAZ connected time=1713529966412
J21TMreKF0ZaW8yxAAAb connected time=1713529968716
cukeSy0oPOEJ32tiAAAd connected time=1713529971408
3n-qnwGNalN1-ovwAAAf connected time=1713554129347
root@Unraid-1:~#
```

**Comment by j0nnymoe** [comment]
`Used the Unraid App Store image` is not good enough. We want the docker run command.

**Comment by servebeer** [comment]
Not exactly sure if there is a way to pull that from Unraid. Here are screenshots of the configuration screen that I assume are used when starting the image:

![Screenshot 2024-04-19 at 14 57 30](https://github.com/linuxserver/docker-netbootxyz/assets/8514341/fcc9615f-7c8d-43ed-8b44-7a1a1a8c43a7)

![Screenshot 2024-04-19 at 14 58 15](https://github.com/linuxserver/docker-netbootxyz/assets/8514341/e2e68cba-0fe0-4ab8-ab44-6ce902f3c281)

![Screenshot 2024-04-19 at 14 58 56](https://github.com/linuxserver/docker-netbootxyz/assets/8514341/183a8cc1-1eba-483a-b3c4-d22128dce0b6)

**Comment by servebeer** [comment]
I think something might have changed in a recent update, as I have never seen these errors before:

![Screenshot 2024-04-19 at 15 07 17](https://github.com/linuxserver/docker-netbootxyz/assets/8514341/12b75647-f4ef-4267-bfe8-7c10de3c54cb)

**Comment by servebeer** [comment]
Just found the Docker Run command:

docker run
  -d
  --name='netbootxyz'
  --net='br0'
  --ip='192.168.100.222'
  -e TZ="America/Chicago"
  -e HOST_OS="Unraid"
  -e HOST_HOSTNAME="Unraid-1"
  -e HOST_CONTAINERNAME="netbootxyz"
  -e 'TCP_PORT_3000'='3000'
  -e 'UDP_PORT_69'='69'
  -e 'TCP_PORT_80'='8084'
  -e 'MENU_VERSION'='1.9.9'
  -e 'PORT_RANGE'='30000:30010'
  -e 'SUBFOLDER'='/'
  -e 'WEB_APP_PORT'='3000'
  -e 'PUID'='99'
  -e 'PGID'='100'
  -e 'UMASK'='022'
  -l net.unraid.docker.managed=dockerman
  -l net.unraid.docker.webui='http://[IP]:[PORT:3000]'
  -l net.unraid.docker.icon='https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/netbootxyz-logo.png'
  -v '/assets':'/assets':'rw'
  -v '/mnt/user/appdata/netbootxyz':'/config':'rw' 'lscr.io/linuxserver/netbootxyz:latest'

812159c58670f4130dcf2bad163497cfe2d6a2e09808d24191bd494414d983f2

The command finished successfully!

**Comment by j0nnymoe** [comment]
Are you trying to use a VM on unraid to boot from netboot?

**Comment by servebeer** [comment]
@j0nnymoe yes, I just fixed the issue. I mannually created the server-info dirctory and favicon.ico file. That in itself did not fix the issue. I then had to change the boot file from netboot.xyz.efi  to netboot.xyz.kpxe. The efi file worked before when everything was fine (I don't use this very often, so it has been a couple of months since I tried to use it). Anyway... it appears that it is working fine now, thanks!
